### PR TITLE
Setup Testing

### LINKED ISSUE #32: Testing setup
We are not going to do a massive amount of tests it's mainly for the calculator logic all that we need to do is: 

- [x] Connect travis with our repo so it woud run on every PR
- [x] install tape and create a basic test in a folder called tests 
- [x] run the tests in the pre commit

PS : this might involve creating a file called travis.yml so you could run travis on github check it out `travis.yml`

### PR DESCRIPTION
**Pull request by SamiSha99**
close #32 

I'm gonna start adding tests soon.

### CODE REVIEW

**Review by Amirk390 — APPROVED** [review]
Looks ok

**Review by Amirk390 — APPROVED** [review]
:100:

File: tests/test.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import test from 'tape';
+
+// var test = require('tape');
```
Comment (matthewdking) [inline tests/test.jsx]: can remove this :wastebasket:

File: tests/test.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+test('Testing Travis', (t) => {
```
Comment (matthewdking) [inline tests/test.jsx]: to be annoying this test is actually testing value of 1

File: package.json [hunk 3]
```diff
@@ -30,11 +34,13 @@
     "react-dom": "^16.2.0",
     "react-router-dom": "^4.2.2",
     "styled-components": "^3.1.6",
+    "tape": "^4.8.0",
```
Comment (matthewdking) [inline package.json]: this should be a devDependency

File: tests/test.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import test from 'tape';
```
Comment (matthewdking) [inline tests/test.jsx]: this is the test file? It doesn't include `jsx` therefore I would have it as a `js` file

**Review by matthewdking — CHANGES_REQUESTED** [review]
yay can't wait to see some tests :sparkles:

File: src/fixtures/data.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+module.exports = {
```
Comment (matthewdking) [inline src/fixtures/data.jsx]: Didn't spot this before but this file was merged in the previous PR so shouldn't be in this one as well?

**Review by matthewdking — CHANGES_REQUESTED** [review]

**Review by matthewdking — APPROVED** [review]
I would remove that test once you have a more useful one implemented :+1:

**Pull request merged**